modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/store/getValues.es.js

```diff
@@ -0,0 +1,38 @@
+import {isString} from 'metal';
```

Comment:
We don't need this file, just use `fetch`. Import fetch with relative path from this module.